### DISCUSSION TITLE
Display credit for themes in the documentation

### DISCUSSION THREAD

**Discussion by jos3s** [JanDeDobbeleer/oh-my-posh#579] [Ideas]
Display below each theme image the name of the Theme creator, with the link to your github profile. It would be possible?

[Documentation](https://ohmyposh.dev/docs/themes)

**Comment by JanDeDobbeleer** [comment]
Sure. Can be done if that matters to you ☺️

**Reply by jos3s** [reply]
Also, is there any way to update the themes page automatically? Since that page is a little out of date at the moment

**Reply by JanDeDobbeleer** [reply]
No. That was manual work and personally I'm not in favor of it's existence. Because of what you just said. There might be a solution though.

**Reply by tradiff** [reply]
I've played around with automating this a little, but I didn't get very far.  Basically I think it could be a Go app to open a terminal window, send keys to it, and take a screenshot. Docker could probably be involved in the terminal that gets opened, to ensure it's always a consistent environment, not just whoever's laptop it happens to be running on.  It's a lot of work though 😅

**Reply by JanDeDobbeleer** [reply]
I have a way, but it fails for colors that aren't standard base16. Pretty sure I could fork the tool and extend it. See [here](https://github.com/JanDeDobbeleer/oh-my-posh/pull/594).

![test](https://user-images.githubusercontent.com/2492783/112763812-58443280-9006-11eb-966f-018e8cfe8803.png)

**Reply by JanDeDobbeleer** [reply]
I found a new [tool](https://github.com/homeport/termshot) which seems to do what we want but fails to render background colors. I'll check if I can "fix" that.

**Reply by JanDeDobbeleer** [reply]
Progress...
![image](https://user-images.githubusercontent.com/2492783/113509754-9639dd00-9557-11eb-984d-57b1d34e9ee4.png)

**Reply by JanDeDobbeleer** [reply]
Almost there. You can follow along [here](https://github.com/JanDeDobbeleer/oh-my-posh/pull/613).

![image](https://user-images.githubusercontent.com/2492783/113518460-ccdb1c00-9586-11eb-8022-d17670550177.png)

**Reply by jos3s** [reply]
Looking forward to it
